src/main.cpp

```diff
@@ -45,21 +45,20 @@ int main(int argc, char **argv) {
   App.set_config("-c,--config_file", "", "Read configuration from an ini file",
                  false);
 
-  // setup logger
   auto Logger = spdlog::stderr_color_mt("LOG");
-  Logger->info("Welcome to spdlog!");
 
   CLI11_PARSE(App, argc, argv);
-  std::string ErrStr;
-  auto KafkaConnection = std::make_unique<ConnectKafka>(Broker, ErrStr);
 
+  auto KafkaConnection = std::make_unique<ConnectKafka>(Broker);
   RequestHandler NewRequestHandler(std::move(KafkaConnection), UserArguments);
   try {
     NewRequestHandler.checkAndRun();
   } catch (ArgumentsException &E) {
-    std::cout << E.what() << std::endl;
+    Logger->error(E.what());
+  } catch (std::runtime_error &E) {
```

Comment:
Same as earlier, `std::runtime_error` would also be caught by `catch (std::exception &E)` and you aren't handling them any differently.